client/store/planner.js

```diff
@@ -106,46 +105,42 @@ export const actions = {
     }
     storeToLocalStorage(state)
   },
-  async addCourse ({ commit, dispatch }, { payload, $api }) {
+  async addCourse({ commit, dispatch }, { payload, $api }) {
     await commit('addCourse', payload)
     await dispatch('saveSettings', { $api })
   },
-  async removeCourse ({ commit, dispatch }, { payload, $api }) {
+  async removeCourse({ commit, dispatch }, { payload, $api }) {
     await commit('removeCourseIndex', payload)
     await dispatch('saveSettings', { $api })
   },
-  async addPlan ({ commit, dispatch }, { payload, $api }) {
+  async addPlan({ commit, dispatch }, { payload, $api }) {
     await commit('addPlan', payload)
     await dispatch('saveSettings', { $api })
   },
-  async setTitleOf ({ commit, dispatch }, { payload, $api }) {
+  async setTitleOf({ commit, dispatch }, { payload, $api }) {
     await commit('setTitleOf', payload)
     await dispatch('saveSettings', { $api })
   },
-  async setCurPlan ({ commit, dispatch }, { payload, $api }) {
+  async setCurPlan({ commit, dispatch }, { payload, $api }) {
     await commit('setCurPlan', payload)
     await dispatch('saveSettings', { $api })
   },
-  async removePlan ({ commit, dispatch }, { payload, $api }) {
+  async removePlan({ commit, dispatch }, { payload, $api }) {
     await commit('removePlan', payload)
     await dispatch('saveSettings', { $api })
   },
-  async loadSettings (
-    {
-      commit,
-      rootState: {
-        auth: { loggedIn, user }
-      }
-    },
-    { $api }
-  ) {
-    const savedPlans = ls('plans')
-    const savedPlan = ls('plan')
+  async loadSettings({
+    commit,
+    rootState: { auth: { loggedIn, user } }
+  }, { $api }) {
+    const savedPlans = ls('plans'),
+    savedPlan = ls('plan')
     if (loggedIn) {
       const { plans, plan } = await $api.auth.getPlans()
-      await commit('setPlans', plans || savedPlans || DEFAULT_PLANS)
+      await commit('setPlans', plans || savedPlan || DEFAULT_PLANS)
```

Comment:
This bit looks like it was correct, not sure why you changed it?

Reply:
I am also not sure why this was changed, I'll change it back. It was probably done when I did ctrl + f to change another variable and this got caught in it.

Reply:
I also do not know why I changed it, probably accidentally when changing something else named "plans." I changed it back